src/core/foundations/src/typography/fs.ts

```diff
@@ -25,8 +40,7 @@ export const fs: Fs = category => (
 	// font is unavailable
 	const requestedFont = availableFonts[category][fontWeight]
 	const fontWeightValue = requestedFont ? fontWeightMapping[fontWeight] : ""
-	const fontStyleValue =
-		italic && requestedFont && requestedFont.hasItalic ? "italic" : ""
+	const fontStyleValue = getFontStyle(requestedFont, fontStyle)
```

Comment:
I like this. Perhaps all of this `xxxValue` logic should live in separate functions.

Reply:
Functions 💪